index.html

```diff
@@ -4479,6 +4479,50 @@ <h2>HTTP</h2>
         <section class="informative">
           <h2>Credential Type-Specific Processing</h2>
 
+          <p>
+RDF is a technology for modeling graphs of statements. Each statement is a
+single <i>subject&#x2192;property&#x2192;value</i> (also known as
+<i>entity&#x2192;attribute&#x2192;value</i>) relationship, which is referred to
+as a <a>claim</a> in this specification. JSON-LD is a technology that enables
+the expression of RDF using idiomatic JSON, enabling developers familiar with
+JSON to write applications to consume RDF as JSON. In general, subjects are
+expressed as JSON objects with each property and value of the subject as a JSON
+key and value, respectively. A special affordance is made to express an
+identifier of a subject, if present, using the <code>@id</code> JSON key, which
+is aliased in this specification to <code>id</code>. See
+<a data-cite="?JSON-LD11#relationship-to-rdf">Relationship to RDF</a> for more
+details.
+          </p>
+
+          <p>
+As JSON can be used to express all different kinds of information, a consumer of
+a particular JSON document can only properly interpret the authors intent if they
+possess information that contextualizes and disambiguates it from other possible
+expressions. Information to assist with this interpretation can either be wholly
+external to the JSON document or linked from within it. Compact JSON-LD documents
+include a <code>@context</code> property that internally expresses or links to
+contextual information and, per the JSON-LD specification, these documents follow
+object-oriented organizational principles to express <a>claims</a>. These features
+enable generalized processors to be written to convert JSON-LD documents from one
+context to another, but this is not needed when consumers receive JSON-LD documents
+that already use the context and shape that they expect. Authors of JSON-LD
```

Comment:
The whole issue/information about generalized processors converting JSON-LD documents seems superfluous to me altogether. Average VC application developers do not even understand what you refer to at this point. I believe the whole sentence can be safely deleted.

Reply:
I won't lie down the road on this, it is an informational text anyway.